main.tf

```diff
@@ -51,17 +53,20 @@ module "eks_blueprints" {
 }
 
 module "eks_blueprints_kubernetes_addons" {
-  source = "github.com/aws-ia/terraform-aws-eks-blueprints//modules/kubernetes-addons?ref=v4.16.0"
+  #source = "github.com/aws-ia/terraform-aws-eks-blueprints//modules/kubernetes-addons?ref=v4.17.0"
+  source = "github.com/allamand/terraform-aws-eks-blueprints//modules/kubernetes-addons?ref=appmesh-prometheus"
```

Comment:
This would need to be change once the PR is finished and merge, to have the prometheus-appmesh addon